client/javascript/views/addpatient.ejs

```diff
@@ -79,26 +104,73 @@
         <option value="AB-">AB-</option>
       </select>
     </div>
+    </div>
+    <div class="row">
+    <div class="col-md-2">
+      <label for="name" class="form-label">Age</label>
+      <input type="text" class="form-control" name="age" required>
```

Comment:
Make a provision that age must be within range of >1 to <100

Reply:
doing it 5 to 70

Reply:
fine